### PR TITLE
added private FQDN

### PR DESCRIPTION
**Pull request by weinong**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by xizhamsft** [comment]
LGTM

**Review by xizha162 — APPROVED** [review]

**Comment by weinong** [comment]
@sergey-shandar can you review this PR? I think I need one more?

**Review by sergey-shandar — APPROVED** [review]

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-10-01/managedClusters.json [hunk 1]
```diff
@@ -1772,6 +1772,11 @@
           "type": "string",
           "description": "FQDN for the master pool."
         },
+        "privateFQDN": {
```
Comment (sergey-shandar) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-10-01/managedClusters.json]: Please, add an example how to use the property.

Comment (weinong) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-10-01/managedClusters.json]: @sergey-shandar added

**Comment by weinong** [comment]
@KrisBash are you the right person to review from ARM side?

**Comment by sauryadas** [comment]
@ravbhatnagar Please help with this review. This is work scheduled for Ignite.

**Review by KrisBash — APPROVED** [review]
LGTM

**Comment by weinong** [comment]
@sergey-shandar can you help merge it?

**Comment by weinong** [comment]
@sergey-shandar I removed the example as it will be a [duplicate key](specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-10-01/examples/ManagedClustersEnablePrivateClusterGet.json) in swagger. After all, it's the same API as ManagedClusters_Get. Just a variant feature.

**Comment by weinong** [comment]
@jhendrixMSFT would you be able to help merge it while Sergey is OOF please?

**Pull request merged**